Assets/Scripts/Player/PlayerController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerController : MonoBehaviour
+{
+    public float moveSpeed = 1;
+ 
+    // Start is called before the first frame update
+    void Start()
+    {
+
+    }
+
+    // FixedUpdate is called at a fixed interval independently of frame rate.
+    void FixedUpdate()
+    {
+        MovePlayer();
+    }
+
+    /// <summary>
+    /// To be executed in update. Moves the player according to current input.
+    /// </summary>
+    void MovePlayer()
+    {
+        // Read player input
+        float moveHorizontal = Input.GetAxis("Horizontal");
+        float moveVertical = Input.GetAxis("Vertical");
+
+        // Calculate new movement vector
+        Vector3 movement = new Vector3(moveHorizontal, 0, moveVertical);
+
+        // Move the player object
+        transform.Translate(movement * moveSpeed * Time.deltaTime);
```

Comment:
Is `Time.deltaTime` required if this method is only called from `FixedUpdate()`?

Reply:
Not 100% sure but seems like the rate can be chosen so it's still need to accommodate for that https://www.reddit.com/r/Unity3D/comments/fgny4v/timedeltatime_in_fixedupdate1/